vcs/repository.go

```diff
@@ -130,6 +133,14 @@ type CommitsOptions struct {
 	NoTotal bool // avoid counting the total number of commits
 }
 
+// CommittersOptions specifies limits on the list of committers returned by
+// (Repository).Committers.
+type CommittersOptions struct {
+	N int // limit the number of returned committers, ordered by decreasing number of commits (0 means no limit)
+
+	Rev string // the rev for which committer stats will be fetched ("" means HEAD)
```

Comment:
Does `HEAD` have meaning in general VCS terms, or is it git-specific?

Have you considered making this parameter mandatory, like `Head` is in `CommitsOptions`? Would that be a better or worse API?